mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -119,3 +127,4 @@ CLASS zq net/minecraft/util/math/MathHelper
 	METHOD i fastInverseSqrt (D)D
 		ARG 0 value
 	METHOD j ease (D)D
+	METHOD k sgn (D)I
```

Comment:
sign?

Reply:
I think `sgn` is a [more common name for the sign function](https://en.m.wikipedia.org/wiki/Sign_function), the same way we have `sin` and `cos` rather than `sine` and `cosine`. But that's just my opinion, I don't mind changing it if you disagree.

Reply:
Ah, thought it was a typo. Didn't realise `sgn` is commonly used

Reply:
`sgn` seems needlessly obfuscated to me, but if it is actually common it may not be too big of a deal. I would note however that the JDK equivalent is called `signum`.

Reply:
`sgn` is the common mathematical short name for that function. `signum` or `sign` is the long name.

https://en.wikipedia.org/wiki/Sign_function